docs/setup/install.md

```diff
@@ -0,0 +1,29 @@
+# Installation
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
```

Comment:
I think the link to how to install kubernetes tools should be here.

Maybe we can change the rest of the docs to match.